src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/TilsynBarnBeregningService.kt

```diff
@@ -1,11 +1,157 @@
 package no.nav.tilleggsstonader.sak.vedtak.barnetilsyn
 
+import no.nav.tilleggsstonader.kontrakter.felles.alleDatoer
+import no.nav.tilleggsstonader.kontrakter.felles.erSortert
+import no.nav.tilleggsstonader.kontrakter.felles.overlapper
+import no.nav.tilleggsstonader.kontrakter.felles.splitPerMåned
+import no.nav.tilleggsstonader.libs.utils.VirkedagerProvider
+import no.nav.tilleggsstonader.sak.infrastruktur.exception.feilHvis
+import no.nav.tilleggsstonader.sak.infrastruktur.exception.feilHvisIkke
 import org.springframework.stereotype.Service
+import java.math.BigDecimal
+import java.math.RoundingMode
+import java.time.YearMonth
+import java.util.UUID
+
+private val SATS_PROSENT = BigDecimal("0.64")
```

Comment:
Skulle ikke denne bytte navn til `DEKNINGSGRAD` da kan man bruke _faktisk dekningsgrad_ for å omtale prosenten av utgiften bruker faktisk får dekt. 

Evt. kunne vi vurdert `FULL_DEKNINGSGRAD` og bruke _dekningsgrad_ for å omtale prosentent av utgiften som faktisk blir dekt. 

Prosenten som faktisk blir dekt er ikke noe jeg tror vi trenger å forholde oss til i koden, men brukes en del i samtaler